### PR TITLE
Option to hide reusability overlays.

### PR DESCRIPTION
**Pull request by rpiaggio**
Kapture 2022-05-14 at 16 27 53

### CODE REVIEW

**Review by toddburnside — APPROVED** [review]
Nice. Will eliminate the need for me to comment out the overlays in the scala code all the time. Thanks.

File: explore/src/main/scala/explore/TopBar.scala [hunk 1]
```diff
@@ -177,19 +178,25 @@ object TopBar {
...
                       // DELETME
-                      DropdownItem(text = "Reset Local Obs",
-                                   icon = Icons.SkullCrossBones.fixedWidth(),
-                                   onClick = ExploreLocalPreferences.cleanObsConfig[IO].runAsync
+                      DropdownItem(
+                        text = "Reset Local Obs",
+                        icon = Icons.SkullCrossBones.fixedWidth(),
+                        onClick = ExploreLocalPreferences.cleanObsConfig[IO].runAsync
+                      )
+                        .when(appCtx.environment === ExecutionEnvironment.Development),
```
Comment (cquiroz) [inline explore/src/main/scala/explore/TopBar.scala:195]: Only dev, great

**Review by cquiroz — APPROVED** [review]
This is going to be useful, thanks

**Comment by cquiroz** [comment]
Maybe we should hide the indicator by default and when needed activate it?

**Comment by toddburnside** [comment]
> 



> Maybe we should hide the indicator by default and when needed activate it?

That would work for me.

**Pull request merged**